app/services/email_service/jobs/request_email_verification.rb

```diff
@@ -0,0 +1,25 @@
+module EmailService::Jobs
+  class RequestEmailVerification < Struct.new(:community_id, :id)
+
+    AddressStore = EmailService::Store::Address
+
+    include DelayedAirbrakeNotification
+
+    def perform
+      if ses_client
```

Comment:
Yeah. It's an error to schedule this job without ses_client being configured. It's a different process though so it might not be configured here but in that case it would be better to get an error raised to detect it than let if fail silently. I'll remove the check.